src/Schema/Events/Click.ts

```diff
@@ -522,6 +521,37 @@ export interface ClickedNavigationTab {
   subject: string
 }
 
+/**
+ *  User clicks in one of the price options on the offer page
+ *
+ *  This schema describes events sent to Segment from [[clickedOfferOption]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOfferOption",
+ *    flow: Make Offer
```

Comment:
This is a string so you need `"`: "Make Offer"
You also need a comma at the end of the line.